packages/integrations/cloudflare/README.md

```diff
@@ -147,38 +146,80 @@ If you want to use the automatic `_routes.json` generation, but want to exclude
 
 The following example automatically generates `_routes.json` while including and excluding additional routes. Note that that is only necessary if you have custom functions in the `functions` folder that are not handled by Astro.
 
-```diff
+```js ins={5-9}
 // astro.config.mjs
 export default defineConfig({
     adapter: cloudflare({
         mode: 'directory',
-+       routes: {
-+           strategy: 'include',
-+           include: ['/users/*'], // handled by custom function: functions/users/[id].js
-+           exclude: ['/users/faq'], // handled by static page: pages/users/faq.astro
-+       },
+        routes: {
+            strategy: 'include',
+            include: ['/users/*'], // handled by custom function: functions/users/[id].js
+            exclude: ['/users/faq'], // handled by static page: pages/users/faq.astro
+        },
     }),
 });
 ```
 
-## Enabling Preview
+### `wasmModuleImports`
+
+`wasmModuleImports: boolean`
 
-In order for preview to work you must install `wrangler`
+default: `false`
 
-```sh
-pnpm install wrangler --save-dev
+Whether or not to import `.wasm` files [directly as ES modules](https://github.com/WebAssembly/esm-integration/tree/main/proposals/esm-integration) using the `.wasm?module` import syntax.
+
+Add `wasmModuleImports: true` to `astro.config.mjs` to enable this functionality in both the Cloudflare build and the Astro dev server. [Read more](#use-wasm-modules)
+
+```js ins={7}
+// astro.config.mjs
+import {defineConfig} from "astro/config";
+import cloudflare from '@astrojs/cloudflare';
+
+export default defineConfig({
+    adapter: cloudflare({
+       wasmModuleImports: true
+    }),
+    output: 'server'
+})
 ```
 
-It's then possible to update the preview script in your `package.json` to `"preview": "wrangler pages dev ./dist"`. This will allow you to run your entire application locally with [Wrangler](https://github.com/cloudflare/wrangler2), which supports secrets, environment variables, KV namespaces, Durable Objects and [all other supported Cloudflare bindings](https://developers.cloudflare.com/pages/platform/functions/#adding-bindings).
+### `runtime`
+
+`runtime: "off" | "local" | "remote"`
+
+default `"off"`
+
+Determines whether and how the Cloudflare Runtime is added to `astro dev`. The Cloudflare Runtime includes [Cloudflare bindings](https://developers.cloudflare.com/pages/platform/functions/bindings), [environment variables](https://developers.cloudflare.com/pages/platform/functions/bindings/#environment-variables), and the [cf object](https://developers.cloudflare.com/workers/runtime-apis/request/#incomingrequestcfproperties). [Read more](#access-to-the-cloudflare-runtime)
```

Comment:
```suggestion
Determines whether and how the Cloudflare Runtime is added to `astro dev`. 

The Cloudflare Runtime includes [Cloudflare bindings](https://developers.cloudflare.com/pages/platform/functions/bindings), [environment variables](https://developers.cloudflare.com/pages/platform/functions/bindings/#environment-variables), and the [cf object](https://developers.cloudflare.com/workers/runtime-apis/request/#incomingrequestcfproperties). Read more about [accessing the Cloudflare Runtime](#access-to-the-cloudflare-runtime).
```
Editing trick! Always imagine a screen reader reading out each link text when you're at the top of the page: it should be exactly clear where each link takes you/why you'd want to go there!